src/ui/mod.rs

```diff
@@ -875,10 +875,10 @@ impl UIElement for Image {
                 r.y,
                 r.w,
                 r.h,
-                self.texture_coords.0,
-                self.texture_coords.1,
-                self.texture_coords.2,
-                self.texture_coords.3,
+                self.texture_coords.0 / 256.0,
```

Comment:
Is there any chance we can describe where this magical value comes from?

Reply:
I looked through some of the rendering code but i dont know why these need to be divided by 256. I just didnt want to write "/ 256.0" every time i set the texture_coords of something.